app/[locale]/(interview)/interview/_components/InterviewShell.tsx

```diff
@@ -0,0 +1,31 @@
+import { getStage } from '~/server/queries/stages';
+import Navigation from './Navigation';
+import StageShell from './StageShell';
+import type { Interview } from '@prisma/client';
+
+// This is the shell for the interview. It contains the navigation and the current stage shell.
+export default async function InterviewShell({
+  interview,
+  currentStage,
+}: {
+  interview: Interview;
+  currentStage: number;
+}) {
+  const isReadyForNextStage = false;
+  const progress = 0;
```

Comment:
What do you think about implementing `progess` as `const progress = stage?.id ?? 0;` demonstration? I think it would be nice if you see some progress when navigating. Please ignore this if you feel this is unnecessary.

Reply:
Great point - thanks! I went ahead and implemented a starter for the progress bar, based on the implementation in Fresco, and replaced the number there with that. I calculated the progress based on the current stage and stage count. Let me know what you think [e62270f](https://github.com/complexdatacollective/Studio/pull/30/commits/e62270f78566d04e642bcb5a127fddb08999f10d)